packages/app/src/components/FeeData.vue

```diff
@@ -86,7 +84,7 @@ getTokenInfo(currentNetwork.value.baseTokenAddress);
 
 const initialFee = computed(() => {
   if (props.feeData) {
-    return BigNumber.from(props.feeData.amountPaid).add(props.feeData.amountRefunded).toHexString();
+    return `0x${(BigInt(props.feeData.amountPaid) + BigInt(props.feeData.amountRefunded)).toString(16)}`;
```

Comment:
Same as previous, I think you should just use toHex utility from ethers

Reply:
Ethers v6 doesn't provider a functiont transform a bigint to a hex string. Added an util function and used it in places like this.